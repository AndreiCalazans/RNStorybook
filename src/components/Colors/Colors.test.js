import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  storyKindRegex: /^Colors$/,
});
