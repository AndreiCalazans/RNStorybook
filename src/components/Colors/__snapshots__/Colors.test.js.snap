// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Colors default 1`] = `
<RCTScrollView
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingBottom": 10,
        "paddingLeft": 10,
        "paddingRight": 10,
        "paddingTop": 10,
      },
      undefined,
    ]
  }
>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      color="black"
      css={
        Object {
          "fontFamily": "Helvetica",
          "letterSpacing": -1.5,
        }
      }
      disabled={false}
      ellipsizeMode="tail"
      fontSize={22}
      style={
        Array [
          Object {
            "color": "black",
            "fontFamily": "Helvetica",
            "fontSize": 22,
            "letterSpacing": -1.5,
          },
          undefined,
        ]
      }
    >
      Color stories
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#000000",
          },
        ]
      }
    >
      black
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#2B2B2B",
          },
        ]
      }
    >
      darkGrey
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#D5D5D5",
          },
        ]
      }
    >
      darkGreyOff
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#FF4F4F",
          },
        ]
      }
    >
      inlineError
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#F5A623",
          },
        ]
      }
    >
      inlineWarning
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#959595",
          },
        ]
      }
    >
      lightGrey
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#000000",
          },
        ]
      }
    >
      offDIAMOND
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#725313",
          },
        ]
      }
    >
      offGOLD
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#3C3E3F",
          },
        ]
      }
    >
      offSILVER
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#A35C05",
          },
        ]
      }
    >
      offSMILES
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#FF5A00",
          },
        ]
      }
    >
      orange
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#ACACAC",
          },
        ]
      }
    >
      separator
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "transparent",
          },
        ]
      }
    >
      transparent
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#FFFFFF",
          },
        ]
      }
    >
      white
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#EFEFEF",
          },
        ]
      }
    >
      greyPlaceholder
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#FF592E",
          },
        ]
      }
    >
      outrageousOrange
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#479BCF",
          },
        ]
      }
    >
      shakespeare
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#916EA3",
          },
        ]
      }
    >
      trendyPink
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#009EA8",
          },
        ]
      }
    >
      bondiBlue
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#5E9BC8",
          },
        ]
      }
    >
      skyblue
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#F7B917",
          },
        ]
      }
    >
      buttercup
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#FA6447",
          },
        ]
      }
    >
      autumn
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#F1F1F1",
          },
        ]
      }
    >
      generic
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#5C5F9B",
          },
        ]
      }
    >
      music
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#608E4C",
          },
        ]
      }
    >
      soccer
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#009BA7",
          },
        ]
      }
    >
      summer
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#2E9A70",
          },
        ]
      }
    >
      voley
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#959595",
          },
        ]
      }
    >
      winter
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#BB3E3E",
          },
        ]
      }
    >
      elderly
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#509AC7",
          },
        ]
      }
    >
      kids
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "height": 20,
            "width": 80,
          },
          Object {
            "backgroundColor": "#8C709E",
          },
        ]
      }
    >
      pets
    </Text>
  </View>
</RCTScrollView>
`;
